pkg/api-server/globalcp_ws.go

```diff
@@ -0,0 +1,14 @@
+package api_server
+
+import (
+	"github.com/emicklei/go-restful"
+
+	globalcp "github.com/Kong/kuma/pkg/globalcp/server"
+)
+
+func globalcpWs(globalcp globalcp.GlobalCP) *restful.WebService {
+	ws := new(restful.WebService).Path("/globalcp")
```

Comment:
how about `/status/control-planes` or `/status/local-control-planes` instead of `/globalcp`

Reply:
`/status/clusters` for consistency with naming.